docs/updating/migration-to-cdn/vuejs-v3.md

```diff
@@ -0,0 +1,189 @@
+
+Replace the CKEditor&nbsp;5 NPM package imports with the CDN script imports and use the `useCKEditorCloud` function to load the CKEditor&nbsp;5 scripts. The `useCKEditorCloud` function is a part of the `@ckeditor/ckeditor5-vue` package and is used to load CKEditor&nbsp;5 scripts from the CKEditor Cloud service.
+
+**Before:**
+
+```html
+<template>
+	<h2>Using CKEditor 5 from NPM in Vue 3</h2>
+
+	<ckeditor
+		v-model="data"
+		tag-name="textarea"
+		:editor="TestEditor"
+		:config="config"
+	/>
+</template>
+
+<script setup lang="ts">
+import { ref, reactive } from 'vue';
+import {
+	ClassicEditor,
+	Essentials,
+	Paragraph,
+	Heading,
+	Bold,
+	Italic,
+	type EventInfo
+} from 'ckeditor5';
+
+class TestEditor extends ClassicEditor {
+	static builtinPlugins = [
+		Essentials,
+		Paragraph,
+		Heading,
+		Bold,
+		Italic
+	];
+}
+
+const data = ref( '<p>Hello world!</p>' );
+const config = reactive( {
+	toolbar: [ 'heading', '|', 'bold', 'italic' ]
+} );
+</script>
+```
+
+**After:**
+
+```html
+<template>
+	<h2>Using CKEditor 5 from NPM in Vue 3</h2>
+  	<ckeditor
+		v-if="TestEditor"
+		v-model="data"
+		tag-name="textarea"
+		:editor="TestEditor"
+		:config="config"
+	/>
+</template>
+
+<script setup lang="ts">
+import { ref, reactive } from 'vue';
+import { useCKEditorCloud } from '@ckeditor/ckeditor5-vue';
+
+const cloud = useCKEditorCloud( {
+	version: '{@var ckeditor5-version}'
+} );
+
+const TestEditor = computed( () => {
+	if ( !cloud.data.value ) {
+		return null;
+	}
+
+	const {
+		ClassicEditor: BaseEditor, Paragraph,
+		Essentials, Heading, Bold, Italic
+	} = cloud.data.value.CKEditor;
+
+	return class TestEditor extends BaseEditor {
+		static builtinPlugins = [
+			Essentials,
+			Paragraph,
+			Heading,
+			Bold,
+			Italic
+		];
+	};
+} );
+
+const data = ref( '<p>Hello world!</p>' );
+const config = reactive( {
+	toolbar: [ 'heading', '|', 'bold', 'italic' ]
+} );
+</script>
+```
+
+### Step 3 (Optional): Migrate the CKEditor&nbsp;5 Vue 3+ integration testing suite
+
+If you have any tests that use CKEditor&nbsp;5 objects, you need to update them to use the  `loadCKEditorCloud` function. Here is an example of migrating a test that uses the `ClassicEditor` object:
+
+**Before:**
+
+```javascript
+import { ClassicEditor, ... } from 'ckeditor5';
+
+it( 'ClassicEditor test', () => {
+	// Your test that uses the CKEditor 5 object.
+} );
+```
+
+**After:**
+
+```javascript
+// It may be counterintuitive that in tests you need to use `loadCKEditorCloud` instead of `useCKEditorCloud`.
+// The reason for this is that `useCKEditorCloud` is a composable and can only be used in Vue components,
+// while tests are typically written as functions in testing suites. Therefore, in tests, you should use
+// the `loadCKEditorCloud` function to load CKEditor 5 from the CKEditor Cloud and obtain the necessary
+// CKEditor 5 objects. This allows you to properly test your CKEditor 5 integration without any issues.
```

Comment:
I like the clarification, but it's hard to read such a long comment in the snippet. Maybe we should extract it to an info box? It looks like a good candidate. WDYT? @godai78
```suggestion
// It may be counterintuitive that in tests you need to use `loadCKEditorCloud` instead of `useCKEditorCloud`.
// The reason for this is that `useCKEditorCloud` is composable and can only be used in Vue components,
// while tests are typically written as functions in testing suites. Therefore, in tests, you should use
// the `loadCKEditorCloud` function to load CKEditor 5 from the CKEditor Cloud and obtain the necessary
// CKEditor 5 objects. This allows you to properly test your CKEditor 5 integration without any issues.
```